translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/-16/translation.txt

```diff
@@ -1,3 +1,3 @@
-﻿「きみとのセックス...素敵だったわ」=Sex with you... was wonderful.
+﻿「きみとのセックス……素敵だったわ」=Sex with you... it was wonderful.

```

Comment:
I don't think the `it` is needed. If you see the `...` as a pause it works better without the `it`